src/shell/shell_batch.cpp

```diff
@@ -78,7 +78,8 @@ bool BatchFile::ReadLine(char * line) {
 				//So we continue reading till EOL/EOF
 				if (((cmd_write - temp) + 1) < (CMD_MAXLINE - 1))
 					*cmd_write++ = c;
-			}
+			} else if (c != 10 && c != 13)
+				shell->WriteOut(MSG_Get("SHELL_ILLEGAL_CONTROL_CHARACTER"), c, c);
```

Comment:
Use `\n` instead of 10 and `\r` instead of 13; use K&R formatting style as well (surround this line with braces).